integrations/azure-devops/azure_devops/client/base_client.py

```diff
@@ -0,0 +1,125 @@
+import httpx
+from httpx import HTTPError, Timeout, BasicAuth, Response
+from typing import Any, AsyncGenerator, Optional
+from port_ocean.utils import http_async_client
+from loguru import logger
+
+PAGE_SIZE = 50
```

Comment:
check if its only the default we might want different values per kind

Reply:
Keeping it this way for now, no particular reason I found to change